module/os/linux/zfs/arc_os.c

```diff
@@ -375,6 +385,50 @@ param_set_arc_int(const char *buf, zfs_kernel_param_t *kp)
 
 	return (0);
 }
+
+/* ARGSUSED */
```

Comment:
This function should also be wrapped by `#ifdef CONFIG_MEMORY_HOTPLUG` to avoid an unused function warning when memory hot plugging is disabled.  The arm builder compile failed due to this.

Reply:
thanks